Sources/Views/Markdown.swift

```diff
@@ -33,6 +33,7 @@ public let markdownBlockStyles: Stylesheet =
       <> blockquoteMarkdownStyles
       <> pMarkdownStyles
       <> codeMarkdownStyles
+      <> key("word-break", "break-word")
```

Comment:
decided to just break words on all markdown containers. fixes the problem in a bunch of places at once

Reply:
Nice, thanks for doing this!